auto_py_to_exe/web/js/i18n.js

```diff
@@ -312,6 +322,7 @@ const translationMap = {
         tr: `${applicationName} ye Özel Seçenekler`,
         th: `${applicationName} ตัวเลือกเฉพาะ`,
         fr: 'Options spécifiques',
+        he: 'אפשרויות ייחודיות של ${applicationName}',
```

Comment:
This is not using the correct quotes - `${applicationName}` will not be evaluated

Reply:
- updated